src/test/unit/sync_http/test_sync_hook.py

```diff
@@ -195,6 +195,30 @@ def lambda_test_function():
     assert Configuration.should_report == "123"
 
 
+def test_bad_domains_scrubber(monkeypatch):
+    monkeypatch.setenv("LUMIGO_DOMAINS_SCRUBBER", '["bad json')
+
+    @lumigo_tracer(token="123", should_report=True)
+    def lambda_test_function():
+        pass
+
+    lambda_test_function()
+    assert utils.Configuration.should_report is False
+
+
+def test_domains_scrubber_happy_flow(monkeypatch):
+    @lumigo_tracer(token="123", domains_scrubber=[".*google.*"])
+    def lambda_test_function():
+        return http.client.HTTPConnection(host="www.google.com").send(b"\r\n")
+
+    lambda_test_function()
+    http_events = SpansContainer.get_span().http_spans
+    assert len(http_events) == 1
+    assert "google" in http_events[0].get("info", {}).get("httpInfo", {}).get("host")
+    assert "headers" not in http_events[0]["info"]["httpInfo"]["request"]
+    assert http_events[0]["info"]["httpInfo"]["request"]["body"] == "The data is not available"
+
```

Comment:
Add a test with `domain_scrubber` but send a request to a website that is not supposed to be scrubbed, and make sure the span collected as before (not scrubbed)